play-2.4/swagger-play2/app/play/modules/swagger/PlayApiReader.scala

```diff
@@ -31,13 +27,31 @@ object SwaggerUtils {
   }
 }
 
-class PlayApiReader(val routes: Option[Router]) extends JaxrsApiReader {
+case class RouteEntry(prefix:String,route:PlayRoute){
+  def httpMethod = route.verb.toString
+  def path =  {
+    val path = if (route.path.toString.startsWith("/")){
```

Comment:
add a space before the { for consistency with existing formatting. this comment applies throughout